### PR TITLE
[CloudMinds] 10316-RCU

### PR DESCRIPTION
**Pull request by Kenny-SZ**
[10316-RCU] This LWM2M Object includes RCU(Robotic Control Unit) information and the link to RCU APPs.

### CODE REVIEW

File: 10316.xml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        <Description><![CDATA[RAM information of the RCU.]]></Description>
+      </Item>
+	  <Item ID="7">
+        <Name>RCU ROM Size</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>G</Units>
```
Comment (seanmcilroy29) [inline 10316.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10316.xml [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        <Description><![CDATA[ROM Size of the RCU (expressed in gigabyte).]]></Description>
+      </Item>
+	  <Item ID="8">
+        <Name>RCU ROM Available Size</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>G</Units>
```
Comment (seanmcilroy29) [inline 10316.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10316.xml [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+        <Description><![CDATA[Available ROM Size of the RCU (expressed in gigabyte).]]></Description>
+      </Item>
+	  <Item ID="9">
+        <Name>SD Storage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>G</Units>
```
Comment (seanmcilroy29) [inline 10316.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10316.xml [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        <Description><![CDATA[Total storage of the RCU SD card, for example: 128G.]]></Description>
+      </Item>
+	  <Item ID="10">
+        <Name>SD Available Storage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>G</Units>
```
Comment (seanmcilroy29) [inline 10316.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10316.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10316.xml]: Should be GB, the unit of storage, gigabyte.

File: 10316.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10316.xml]: Should be GB, the unit of storage, gigabyte.

File: 10316.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10316.xml]: Should be GB, the unit of storage, gigabyte.

File: 10316.xml (hunk 4 above)
Comment (Kenny-SZ) [reply 10316.xml]: Should be GB, the unit of storage, gigabyte.

File: 10316.xml (hunk 1 above)
Comment (seanmcilroy29) [reply 10316.xml]: Gb should not be used as they are commonly used to represent other measurements.
See https://tools.ietf.org/html/rfc8428

File: 10316.xml (hunk 2 above)
Comment (seanmcilroy29) [reply 10316.xml]: Gb should not be used as they are commonly used to represent other measurements.
See https://tools.ietf.org/html/rfc8428

File: 10316.xml (hunk 3 above)
Comment (seanmcilroy29) [reply 10316.xml]: Gb should not be used as they are commonly used to represent other measurements.
See https://tools.ietf.org/html/rfc8428

File: 10316.xml (hunk 4 above)
Comment (seanmcilroy29) [reply 10316.xml]: Gb should not be used as they are commonly used to represent other measurements.
See https://tools.ietf.org/html/rfc8428

**Comment by Kenny-SZ** [comment]
Changed % to /100, GB to gigaB

File: 10316.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10316.xml]: Changed GB to gigaB

File: 10316.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10316.xml]: GB to gigaB

File: 10316.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10316.xml]: Changed GB to gigaB

File: 10316.xml (hunk 4 above)
Comment (Kenny-SZ) [reply 10316.xml]: Changed GB to gigaB

**Comment by Kenny-SZ** [comment]
Changed gigaB to GB.

**Pull request merged**